db/entity/DiscordMember.ts

```diff
@@ -6,14 +6,14 @@ import {
   PrimaryGeneratedColumn,
   Index,
 } from "typeorm";
-import { DiscordServer } from "./DiscordServer";
+import { DiscordServerConfig } from "./DiscordServerConfig";
 
 @Entity()
-@Index(["discordMemberId", "discordServerId", "deletedAt"], {
+@Index(["discordMemberId", "DiscordServerConfigId", "deletedAt"], {
```

Comment:
same, please use the case that is already used in the rest of project to be cleaner
```suggestion
@Index(["discordMemberId", "discordServerConfigId", "deletedAt"], {
```